pallets/nft/src/lib.rs

```diff
@@ -18,19 +20,29 @@ mod tests;
 pub type Dna = Vec<u8>;
 pub type Cid = Vec<u8>;
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, Default)]
+pub struct PendingNft<AccountId, ClassId> {
+	account_id: AccountId,
+	class_id: ClassId,
```

Comment:
@korzewski is this the same CID that will be generated from IPFS storage?

Reply:
Nope, `ClassId` and `TokenId` are just indexes. 
`ClassId` = mould ID
`TokenID` = NFT ID

CID wold be added to `ClassData` https://github.com/anmolnetwork/anmol-node/pull/25/files/4c6d5dab6c8036633e2ac81f78fdeda180ea0c26#diff-9340a89ab12e3b4b5b5e36714cb73d3c82979532d77c70c143ab488ef416154bR35

Reply:
Thanks for the clarification!